packages/widgets/src/dns-hole/summary/component.tsx

```diff
@@ -0,0 +1,173 @@
+"use client";
+
+import type { BoxProps } from "@mantine/core";
+import { Box, Card, Center, Flex, Text } from "@mantine/core";
+import { useElementSize } from "@mantine/hooks";
+import {
+  IconBarrierBlock,
+  IconPercentage,
+  IconSearch,
+  IconWorldWww,
+} from "@tabler/icons-react";
+
+import type { RouterOutputs } from "@homarr/api";
+import { clientApi } from "@homarr/api/client";
+import { formatNumber } from "@homarr/common";
+import type {
+  stringOrTranslation,
+  TranslationFunction,
+} from "@homarr/translation";
+import { translateIfNecessary } from "@homarr/translation";
+import { useI18n } from "@homarr/translation/client";
+import type { TablerIcon } from "@homarr/ui";
+
+import type { WidgetComponentProps, WidgetProps } from "../../definition";
+import { NoIntegrationSelectedError } from "../../errors";
+
+export default function DnsHoleSummaryWidget({
+  options,
+  integrationIds,
+}: WidgetComponentProps<"dnsHoleSummary">) {
+  const integrationId = integrationIds.at(0);
+
+  if (!integrationId) {
+    throw new NoIntegrationSelectedError();
+  }
+
+  const [data] = clientApi.widget.dnsHole.summary.useSuspenseQuery(
+    {
+      integrationId,
+    },
+    {
+      refetchOnMount: false,
+      retry: false,
```

Comment:
Why not retry?

Reply:
Because you can retry yourself, if a request fails it will probably have a reason